Content.Server/Tabletop/TabletopBackgammonSetup.cs

```diff
@@ -27,65 +25,47 @@ public override void SetupTabletop(TabletopSession session, IEntityManager entit
             const float boardDistanceX = 1.25f;
             const float pieceDistanceY = 0.80f;
 
-            float getXposition (float distanceFromSide, bool isLeftSide)
+            float getXPosition(float distanceFromSide, bool isLeftSide)
             {
-                return isLeftSide ? -borderLengthX + (distanceFromSide * boardDistanceX)
-                : borderLengthX - (distanceFromSide * boardDistanceX);
+                var pos = borderLengthX - (distanceFromSide * boardDistanceX);
+                return isLeftSide ? -pos : pos;
             }
 
-            float getYPosition (float positionNumber, bool isTop)
+            float getYPosition(float positionNumber, bool isTop)
             {
-                return isTop ? borderLengthY  - (pieceDistanceY * positionNumber)
-                : -borderLengthY  + (pieceDistanceY * positionNumber);
+                var pos = borderLengthY - (pieceDistanceY * positionNumber);
```

Comment:
I think you just swapped the true and false conditions here, shouldn't change anything but might confuse someone in the future

Reply:
You are eagle eyed because I had accidently skipped right over that. Kind of passed me right by that the whole board was flipped but false memories just filled in the gap.

swapped them back around.